styles.css

```diff
@@ -474,6 +474,103 @@
     background-color: #fff;
     color: #5e6c84;
   }
+
+  .modal {
+    position: fixed;
+    width: 90%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background: rgb(0 0 0 / 50%);
+    z-index: 2;
+    display: none;
+    overflow: auto;
+  }
+
+  .modal-content {
+    background: #fff;
+    margin: 20px;
+    width: 90%;
+    height: auto;
+    padding: 8px 24px;
+    border-radius: 4px;
+    z-index: 4;
+  }
+
+  .modal-data {
+    margin-top: 20px;
+  }
+
+  .modal-data h2 {
+    font-weight: bold;
+  }
+
+  .modal-content strong {
+    font-weight: bold;
+    font-size: 25px;
+  }
+
+  .modal-data-tonic {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    width: 300px;
+  }
+
+  .modal-data-tonic img {
+    width: 10px;
+    height: 10px;
+  }
+
+  .modal-image img {
+    width: 100%;
+    margin-top: 25px;
+  }
+
+  .float-modal {
+    float: left;
+    width: 60%;
+    margin-right: 50px;
+  }
+
+  .modal-close {
+    position: absolute;
+    right: 100px;
+    top: 25px;
+    width: 80px;
+    height: 80px;
+  }
```

Comment:
- [ ] Please justify this button to the end to match the design with Figma:

Figma   | Yours
------------- | -------------
 ![image](https://user-images.githubusercontent.com/77038610/151209795-38c86850-2f0d-4fe8-8dd1-316e78104e54.png) | ![image](https://user-images.githubusercontent.com/77038610/151209704-be0f1c8a-7584-46cb-b823-4c2eed0cca7b.png)